config/locales/work.en.yml

```diff
@@ -94,5 +94,6 @@ en:
         representative: "Select the file with media that represents this Work and is used as a thumbnail."
         contained_by: >
           The sub-Collection within the Digital Collections Application to which the digital work belongs. Select the named collection that the work is part of, if applicable. If a desired entry is not available, contact the Digital Collections Librarian. Required if applicable.
-
+        ocr_requested: >
+          We add and remove OCR on a nightly basis. Check this box if you'd like OCR to be created for this work next time the job runs. Likewise, uncheck the box if you want to remove existing OCR.
```

Comment:
Do we actually plan to REMOVE existing OCR as a result of check?  Is this second sentence an implementation detail they may or may not need?

I'm still not really sure about this "OCR requested" verbiage in the first place... it seems weird to say that turning off the request for something means it gets deleted....  But I guess this is good for now?

Reply:
Done!